gtsfm/utils/io.py

```diff
@@ -101,6 +101,19 @@ def save_json_file(
         json.dump(data, f, indent=4)
 
 
+def read_json_file(fpath: Union[str, Path]) -> Any:
```

Comment:
I am surprised we dont have this yet .. I feel like I saw it somewhere.

Reply:
I thought so too, but I grepped the repo and couldnt find it in `master`.